source/doc_versions.py

```diff
@@ -1,9 +1,9 @@
 # The API version strings to display in the documentation
 # COMMON_VERSION is the latest version of any one API, and is considered the overall API version
-COMMON_VERSION = '2016-06-12'
+COMMON_VERSION = '2016-07-01'
```

Comment:
@mikeweaver there had previously been no documentation for this error, so there was nothing to change. In grooming we determined that if I added documentation for that one error, then we would have to document all errors, so we decided against it.

I named the branch to summarize the jira ticket (or in other words, why I'm making these changes in the first place). Should rename it?